app/assets/javascripts/active_admin/qiniu_input/upload_audio.js

```diff
@@ -33,7 +33,7 @@ $(document).ready(function() {
     if ($hints.find('.audio').length > 1) {
       $hints.find('.audio').attr('src', url);
     } else {
-      $hints.html("<audio src=" + url + " controls='controls' preload='auto'></audio>")
+      $hints.html("<audio src='" + url + "' controls='controls' preload='auto'></audio>")
```

Comment:
should use `"` for wrapping DOM attributes values, for example `$hints.html('<audit src="xxx" controls="xxx" ... ')`